Inclui arquivo para instalação do odoo 10.0 com módulos odoo-brasil

Silmar opened the pull request:
Trust-Code em sistemas Linux baseados em Debian.

Silmar committed:
Inclui arquivo para instalação do odoo 10.0 com módulos odoo-brasil
Trust-Code em sistemas Linux baseados em Debian.

danimaribeiro commented:
Consegue colocar no readme instruções de que este arquivo faz todo o processo de uma vez só?

Silmar committed:
Correção de um detalhe, atualização de README.MD
Incluido referência ao fato de haver um instalador automático.

Silmar committed:
Adicionando mais um requerimento.
Trata-se do node-less que pode não estar incluso por padrão em algumas distros.
